documentjs.json

```diff
@@ -46,7 +46,19 @@
 			]
 		},
 		"2.3": {
-			"source": "git://github.com/bitovi/canjs#master",
+			"source": "git://github.com/canjs/canjs#66940b9c9bbafb7b03921cc9109c995c994df839",
```

Comment:
This should still be master (until we release 3.0). Was there a reason this needed to be changed?